src/icons/folderIcons.ts

```diff
@@ -175,6 +175,7 @@ export const folderIcons: FolderTheme[] = [
             { name: 'folder-prisma', folderNames: ['prisma'] },
             { name: 'folder-pipe', folderNames: ['pipe', 'pipes'] },
             { name: 'folder-svg', folderNames: ['svg', 'svgs'] },
+            { name: 'folder-netlify', folderNames: ['.netlify', 'admin'] },
```

Comment:
I think that "admin" is not only used in netlify projects. Everyone who uses an admin folder will then see the netlify folder icon? We should avoid that.